input/fsh/omrs-encounter.fsh

```diff
@@ -0,0 +1,5 @@
+Profile: OpenMRSEncounter
```

Comment:
Am already working on this Resource, You could delete it from the commit. In cases where you need a Reference that is not yet worked on. Just replace it's  with that of  HL7  then later after merging restore the Reference element with a new PR.